src/integration-test/java/com/commercetools/project/sync/ProductSyncWithReferencesIT.java

```diff
@@ -66,18 +77,49 @@
 
 class ProductSyncWithReferencesIT {
 
-  private static final TestLogger syncerTestLogger = TestLoggerFactory.getTestLogger(Syncer.class);
+  private static final TestLogger productSyncerTestLogger =
+      TestLoggerFactory.getTestLogger(ProductSyncer.class);
   private static final TestLogger cliRunnerTestLogger =
       TestLoggerFactory.getTestLogger(CliRunner.class);
+  private static final TestLogger productTypeSyncerTestLogger =
+      TestLoggerFactory.getTestLogger(ProductTypeSyncer.class);
+  private static final TestLogger customerSyncerTestLogger =
+      TestLoggerFactory.getTestLogger(CustomerSyncer.class);
+  private static final TestLogger shoppingListSyncerTestLogger =
+      TestLoggerFactory.getTestLogger(ShoppingListSyncer.class);
+  private static final TestLogger stateSyncerTestLogger =
+      TestLoggerFactory.getTestLogger(StateSyncer.class);
+  private static final TestLogger inventoryEntrySyncerTestLogger =
+      TestLoggerFactory.getTestLogger(InventoryEntrySyncer.class);
+  private static final TestLogger customObjectSyncerTestLogger =
+      TestLoggerFactory.getTestLogger(CustomObjectSyncer.class);
+  private static final TestLogger typeSyncerTestLogger =
+      TestLoggerFactory.getTestLogger(TypeSyncer.class);
+  private static final TestLogger categorySyncerTestLogger =
+      TestLoggerFactory.getTestLogger(CategorySyncer.class);
+  private static final TestLogger cartDiscountSyncerTestLogger =
+      TestLoggerFactory.getTestLogger(CartDiscountSyncer.class);
+  private static final TestLogger taxCategorySyncerTestLogger =
+      TestLoggerFactory.getTestLogger(TaxCategorySyncer.class);
```

Comment:
Are logged events really so important so that we would unit test all of the resources?

Reply:
Actually no, logging assertions is not a good way to go on test cases, the changes show that. 

But to make the work in scope I choose to not replace log assertions with CTP query assertions. 
If you think it's necessary, I could replace those.

Reply:
ok. Please create a ticket and we can close this one.

Reply:
I opened the follow up ticket, CTPI-507